test/jit/test_complex.py

```diff
@@ -99,7 +106,7 @@ def checkCmath(func_name, funcs_template=funcs_template):
                     self.assertEqual(res_python, res_script, msg=msg)
 
         unary_ops = ['log', 'log10', 'sqrt', 'exp', 'sin', 'cos', 'asin', 'acos', 'atan', 'sinh', 'cosh',
-                     'tanh', 'asinh', 'acosh', 'atanh', 'phase']
+                     'tanh', 'asinh', 'acosh', 'atanh', 'phase', 'isinf', 'isnan', 'isfinite']
 
         # --- Unary ops ---
         for op in unary_ops:
```

Comment:
One cool thing I learned about recently: https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests

Using `with self.subTest(op):` here might make it clear which specific test being generated within this one test is failing.

Reply:
wow super useful!